src/content/blog/2024/sarek_paper.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: 'nf-core/sarek paper'
+subtitle: 'Harder, Better, Faster, Stronger'
+pubDate: 2024-04-26T00:13:00+01:00
```

Comment:
shouldn't the date be today's?

Reply:
yeah but when I put date today it put april 24ths on the preview 🤨